debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/Replicator.java

```diff
@@ -502,6 +502,39 @@ protected void readOplog(MongoClient primary) {
             incompleteEventTimestamp = oplogStart;
             incompleteTxOrder = txOrder.getAsLong();
         }
+
+        Bson operationFilter = null;
+        String operations = context.getConnectionContext().config.getString(MongoDbConnectorConfig.SKIPPED_OPERATIONS);
+        if (operations != null) {
+            for (String operation : operations.trim().split(",")) {
```

Comment:
The trim should be applied to the individual parts. It should handle cases like `u, d`.